api/migrations/versions/d7f3adff2c9f_.py

```diff
@@ -0,0 +1,22 @@
+"""empty message
+
+Revision ID: d7f3adff2c9f
+Revises: ('3e7bdff8af80', 'b6a2a69fbb13')
+Create Date: 2020-02-12 11:12:35.018060
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'd7f3adff2c9f'
+down_revision = ('3e7bdff8af80', 'b6a2a69fbb13')
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    pass
+
+
+def downgrade():
+    pass
```

Comment:
@jainesh99 Seems you did a merge migration but we're still missing the dropping of the column itself?

Reply:
@amritpurshotam , thank you spotting this out for me. I have made the changes, please have a look. 

I have done the automatic migration script, but I had to tweak it a bit because the build was failing on the run migration step due to the "deadline" column containing null values. I added a server default and then altered that column

Reply:
@jainesh99 Did you push the code? I can't see it? Unless I wonder if it's cos the PR was closed

Reply:
Just created a PR for it now.